src/homepage/howTo/imap-server.html

```diff
@@ -0,0 +1,250 @@
+                        <li>Bob resolves <code>mx.company.com</code> ip address...</li>
+                        <li>And can establish a connection to <code>mx.company.com</code> to send an email to Alice</li>
+                    </ol>
+
+                    <p>All is needed is a MX entry in domain name resolution pointing to the future IP of your James server.</p>
+
+                    <header class="major">
+                        <h2><b>JAMES architecture</b></h2>
+                    </header>
+
+                    <p>JPA guice docker image relies on an embedded derby database for storing data. Note that Apache Lucene library
+                        is used for email search. A mail queue is implemented on top of an embedded Apache ActiveMQ. Hence James do not need
+                        any external service for being running.
+                    </p>
+
+                    <img src="/images/guice-jpa-architecture-overview.png">
+
+                    <p>
+                        JPA guice offers the following protocols:
+                    </p>
+
+                    <ul class="no-padding">
+                        <li><b>SMTP</b> For receiving emails</li>
+                        <li><b>IMAP</b> For reading emails</li>
+                        <li><b><a href="http://james.apache.org/server/manage-webadmin.html">WebAdmin</a></b> is a REST API allowing you to manage Apache JAMES</li>
+                    </ul>
+
+                    <p>The following protocols are also available:</p>
+
+                    <ul class="no-padding">
+                        <li><b>LMTP</b> local version of SMTP</li>
+                        <li><b>POP3</b> For reading emails. Lacks tests</li>
+                        <li><b>JMX</b> is used by a <a href="http://james.apache.org/server/manage-cli.html">command line</a> for administrating Apache James</li>
+                        <li>And <a href="http://james.apache.org/server/feature-protocols.html">more</a>...</li>
+                    </ul>
+
+                    <header class="major">
+                        <h2><b>Generation of a custom keystore</b></h2>
+                    </header>
+
+                    <p>In order to maintain a good level of privacy and security, James is relying on TLS cryptography
+                    for securing exchanges. We thus need to generate our own personal keystore. Note that this guide do not
+                    cover generating a keystore from SSL certificates. A security exception might be configured on the Mail
+                    User Agent.</p>
+
+                    <p>In order to create a keystore, please run: <code>keytool -genkey -alias james -keyalg RSA -keystore /path/to/james/conf/keystore</code>.
+                    James is configured with a default password <code>james72laBalle</code> (used to read the keystore). However, we will be overriding the
```

Comment:
we should change that to read an env variable, default password doesn't make sense